### PR TITLE
DOCSP-37540/1/9 Limit, Sort, Skip

### PR DESCRIPTION
**Pull request by jordan-smith721**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-37540
https://jira.mongodb.org/browse/DOCSP-37541
https://jira.mongodb.org/browse/DOCSP-37549

Staging - https://preview-mongodbjordansmith721.gatsbyjs.io/pymongo/DOCSP-37540-limit/read/specify-documents-to-return/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/read/specify-documents-to-return.txt [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+      Acquolina Catering
+
+.. note::
+
+   The order in which you call these methods doesn't change the documents
+   that are returned. The driver automatically reorders the calls to apply the
+   sort and skip first and the limit after.
+
+You can also apply ``limit``, ``sort`` and ``skip`` by specifying them as
+options in the ``find()`` method. The following example specifies the
```
Comment (mongoKart) [inline source/read/specify-documents-to-return.txt:213]: S:

```suggestion
You can also limit, sort, and skip results by using parameters of
the ``find()`` method. The following example specifies the
```

**Review by mongoKart — CHANGES_REQUESTED** [review]
Nice job knocking these out together!

File: source/read/specify-documents-to-return.txt [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+      Acquista Trattoria
+      Acquolina Catering
+
+.. note::
+
+   The order in which you call these methods doesn't change the documents
+   that are returned. The driver automatically reorders the calls to apply the
+   sort and skip first and the limit after.
+
+You can also apply ``limit``, ``sort`` and ``skip`` by specifying them as
```
[the remark this answers is not in the document]
Comment (jordan-smith721) [reply source/read/specify-documents-to-return.txt]: Pretty sure my brain must have been translating backticks into commas in this PR

**Review by mongoKart — APPROVED** [review]
LGTM. Nice job!

**Review by blink1073 — APPROVED** [review]
LGTM!

**Pull request merged**